installer/roles/image_build/files/supervisor_task.conf

```diff
@@ -50,7 +38,7 @@ stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
 [group:tower-processes]
-programs=celery,celery-watcher,callback-receiver,channels-worker
+programs=dispatcher,callback-receiver,channels-worker
```

Comment:
@shanemcd we'll have to copy this into wherever it belongs for prod OpenShift builds (is there a similar OpenShift-oriented version of this in `tower-packaging`?)